test/verify/check-storage-stratis

```diff
@@ -151,10 +160,11 @@ class TestStorageStratis(storagelib.StorageCase):
         self.addCleanupMount("/run/fsys2")
         b.wait_text(self.card_row_col("Stratis filesystems", 2, 1), "fsys2")
         b.wait_text(self.card_row_col("Stratis filesystems", 2, 3), "/run/fsys2")
-        # different stratis versions report different usage numbers.
-        b.assert_pixels(self.card("Stratis filesystems"), "fsys-rows",
-                        mock={".usage-text": "0.57 / 7.4 GB"},
-                        ignore=[".usage-bar"])
+        if not legacy:
+            # different stratis versions report different usage numbers.
+            b.assert_pixels(self.card("Stratis filesystems"), "fsys-rows",
+                            mock={".usage-text": "0.57 / 7.4 GB"},
+                            ignore=[".usage-bar"])
```

Comment:
Sounds like we can drop the mocking then

Reply:
But then we need to take the new pixels and with a new stratis the numbers will be different agani, so maybe not :)